testutil/singlevm/setup.sh

```diff
@@ -146,6 +147,8 @@ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout controller_key.pem -
 #Copy the certs
 sudo cp -f controller_key.pem /etc/pki/ciao
 sudo cp -f controller_cert.pem /etc/pki/ciao
+sudo ln -s /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
+sudo ln -s /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
```

Comment:
Doing so would also complicate things I think.  I may be mistaken but I think using separate certs for ciao-image and ciao-controller would require us to create three certs, a cert for controller, a cert for ciao-image and a CA cert that is used to sign the other two certs that can be given to ciao-cli.